src/js/components/wizard.js

```diff
@@ -0,0 +1,110 @@
+import FluxComponent from 'flummox/component';
+import React from 'react';
+
+
+let Wizard = React.createClass({
+  render() {
+    let steps = this.props.children.map(function(step, index) {
+      return React.cloneElement(step, {
+        key: index,
+        isActive: index === this.props.activeStep
+      });
+    }, this);
+
+    return (
+      <section className="wizard">
+        <Wizard.ProgressBar wizard={this}/>
+        {steps}
+        <Wizard.Menu wizard={this}/>
+      </section>
+    );
+  }
+});
+
+
+Wizard.ProgressBar = React.createClass({
+  propTypes: {
+    wizard: React.PropTypes.instanceOf(Wizard).isRequired
+  },
+
+  handleClick(num) {
+    return () => { this.props.wizard.props.flux.getActions('wizard').goToStep(num) }  
+  },
+
+  render() {
+    return (
+      <menu className="wizard--progress-bar">
+        {this.props.wizard.props.children.map(
+          (step, index) => <Wizard.ProgressItem handleClick={this.handleClick(index)} key={index} step={step}/>
```

Comment:
return <menu className="wizard--progress-bar">
      {this.props.wizard.props.children.map(
        (step, index) => 
          <Wizard.ProgressItem handleClick={this.goToStep(index)}
                               key={index} step={step}/>
      )}
    </menu>

I guess this isn't horrible.